src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VurderingRepository.kt

```diff
@@ -166,6 +179,11 @@ class VurderingRepository(private val database: DatabaseInterface) : IVurderingR
                 SELECT * FROM VURDERING WHERE personident=? ORDER BY created_at DESC
             """
 
+        private const val GET_VURDERINGER =
+            """
+                SELECT * FROM VURDERING WHERE personident = ANY (string_to_array(?, ',')) ORDER BY created_at ASC
```

Comment:
Blir spennende! Trenger man `ASC`? Får alltid warning på det i IntelliJ om at det er unødvendig (samme som å skrive `WHEN boolean = TRUE` i stedet for `WHEN boolean`)

Reply:
Når det er vesentlig for at koden skal virke synes jeg det er greit å være overtydelig.